tests/ui/enum_flatten.rs

```diff
@@ -9,10 +9,10 @@
 use structopt::StructOpt;
 
 #[derive(StructOpt, Debug)]
-#[structopt(name = "basic", flatten)]
```

Comment:
This test should not be removed. I think you might add a new test instead of modifying this one.

Reply:
Unfortunately, it's really hard to distinguish between an attr on top of enum variant (valid usage) and on top of struct/enum (invalid usage) in current codebase. I think this test is not so important and the changes required to do it correctly would be quite complex so I propose to ditch this test for now.

Reply:
So this code now compiles? What does it do?

Reply:
Flatten is simply ignored

Reply:
OK, then